### PR TITLE
Add recipe for pytest-ast-back-to-python

### PR DESCRIPTION
**Pull request by nicoddemus**
@tomviner as owner of the package, would you like to be added as maintainer of the conda recipe as well?

### CODE REVIEW

**Comment by tomviner** [comment]
@nicoddemus yes please!

File: recipes/pytest-ast-back-to-python/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  imports:
+    - pytest  # must be imported first because of circular imports in pytest_ast_back_to_python
+    - pytest_ast_back_to_python
+
+about:
+  home: https://github.com/tomviner/pytest-ast-back-to-python
+  license: BSD-3
+  license_family: BSD
```
Comment (jakirkham) [inline recipes/pytest-ast-back-to-python/meta.yaml]: note: No license file yet, but an issue has been raised upstream.

xref: https://github.com/tomviner/pytest-ast-back-to-python/issues/5

**Comment by nicoddemus** [comment]
Added @tomviner as maintainer. 👍

**Comment by nicoddemus** [comment]
I think this is now ready to go. :+1:

**Comment by jakirkham** [comment]
Thanks @nicoddemus.

**Pull request merged**